docs/userguides/projects.md

```diff
@@ -47,6 +47,20 @@ a = accounts.load("metamask_0")
 a.deploy(project.MyContract)
 ```
 
+### Dependencies
+
+To set up dependencies in your ``ape-config.yaml`` file, follow [this guide](https://docs.apeworx.io/ape/stable/userguides/config.html#dependencies).
+If you are setting dependencies from a remote source, they will download when you run `ape compile` or other commands that compile in the background.
+Dependencies only need to download and compile once.
+
+You can access dependency contracts off your root project manager the same as your project's contracts.
+
+```python
+from ape import project
+
+dependency = project.MyDependency
```

Comment:
It might be nicer to use snake_case in the example here, to make it clear it's not a ContractType
```suggestion
dependency = project.my_dependency
```

Reply:
Oh, I need to update these docs completely! I went with the `project.dependencies["my_dependency"].ContractTypeThing` approach because it seemed like you were favoring that yesterday

Reply:
oh yes, I forgot, yes please let's do that